### PR TITLE
Added setup for cloudflared

### PR DESCRIPTION
**Pull request by kensonjohnson**

### CODE REVIEW

File: deploy/setup_cloudflared.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#-----------------------------------------------------------------------------------------------------------#
+# This script sets up a deployment environment on an Ubuntu 20.04/22.04
+#
+# Documentation:
+# https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/
+#
+#-----------------------------------------------------------------------------------------------------------#
+
+
+# Add Cloudflare’s package signing key
```
Comment (christopherkeim) [inline deploy/setup_cloudflared.sh:12]: If we want to integrate these into our setup pipeline we could wrap them to make them idempotent:

```bash
# Add Cloudflare’s package signing key
if [ -f /usr/share/keyrings/cloudflare-main.gpg ]
then
  echo "Cloudflare gpg key installed at /usr/share/keyrings/cloudflare-main.gpg"
else
  sudo mkdir -p --mode=0755 /usr/share/keyrings
  curl -fsSL https://pkg.cloudflare.com/cloudflare-main.gpg | sudo tee /usr/share/keyrings/cloudflare-main.gpg >/dev/null
fi
```

File: deploy/setup_cloudflared.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+sudo mkdir -p --mode=0755 /usr/share/keyrings
+curl -fsSL https://pkg.cloudflare.com/cloudflare-main.gpg | sudo tee /usr/share/keyrings/cloudflare-main.gpg >/dev/null
+
+# Add Cloudflare’s apt repo to your apt repositories
```
Comment (christopherkeim) [inline deploy/setup_cloudflared.sh:16]:
```bash
if [ -f /etc/apt/sources.list.d/cloudflared.list ] 
then
  echo "Cloudfare apt repo already added locally."
else
  echo "deb [signed-by=/usr/share/keyrings/cloudflare-main.gpg] https://pkg.cloudflare.com/cloudflared 
  $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/cloudflared.list
fi

File: deploy/setup_cloudflared.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+echo "deb [signed-by=/usr/share/keyrings/cloudflare-main.gpg] https://pkg.cloudflare.com/cloudflared $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/cloudflared.list
+
+# Update repositories and install cloudflared
```
Comment (christopherkeim) [inline deploy/setup_cloudflared.sh:19]:
```bash
if ( which cloudflared > /dev/null )
then
  echo "cloudflared already installed"
else
  sudo apt-get update && sudo apt-get install cloudflared
fi
```

File: deploy/setup_cloudflared.sh (hunk 1 above)
Comment (kensonjohnson) [reply deploy/setup_cloudflared.sh:12]: The command is already idempotent.

**Review by christopherkeim — APPROVED** [review]

**Pull request merged**